css/application-styles.css

```diff
@@ -0,0 +1,230 @@
+.main-nav {
+  position: static;
+}
+
+.paw-logo {
+  display: inline-block;
+}
+
+/* INPUTS AND LABELS STYLES START */
+input {
+  width:460px;
+  height: 64px;
+  border: 1px solid #D2D2D2;
+  border-radius: 3px;
+  padding: 0;
+  margin-bottom: 20px;
+}
+
+label {
+  font-family: Arial;
+  font-size: 36px;
+  font-weight: 400;
+  line-height: 41px;
+  text-align: left;
+}
+
+/* INPUTS AND LABELS STYLES END */
+
+.grid-container {
+  height: 1012px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.form-applicaion {
+  background: #FFC0CB;
+  border: 1px solid #EDDEDE;
+  border-radius: 12px; 
+  padding: 60px 0 100px 70px;   
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto;
+  grid-template-areas: 'title title'
+                        'labelfirst labelast'
+                        'firstName lastname'
+                        'labelemail labeljobrole'
+                        'email jobrole'
+                        'labeladdress .'
+                        'address address'
+                        'cvlabel .'
+                        'cv . '
+                        'sendForm sendForm';
+  height: 80%;
+}
+
+.form-application-title {
+  grid-area: title;
+  font-family: Arial;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 46px;
+}
+
+.form-applicaion-labelfirstname {
+  grid-area: labelfirst;
+}
+
+.form-applicaion-firstname {
+  grid-area: firstName;
+}
+
+.form-applicaion-labelastname {
+  grid-area: labelast;
+}
+
+.form-applicaion-lastname {
+  grid-area: lastname;
+}
+
+.form-applicaion-labelemail{
+ grid-area: labelemail; 
+}
+
+.form-applicaion-email {
+  grid-area: email;
+}
+
+.form-applicaion-labelfirstname, 
+.form-applicaion-labelastname,
+.form-applicaion-labelemail,
+.form-applicaion-labeladdress,
+.form-applicaion-labeljobrole {
+  align-self: end;
+}
+/* JOB STYLE START */
+
+/* ARROW ON SELECT ELEMENT*/
+
+.dropicon {
+  overflow: hidden;     
+  position: relative;
+}
+
+.dropicon:after{
+  content:"\f078 ";
+  font-family: FontAwesome;
+  position: absolute; 
+  right: 162px; 
+  top: 10px;
+  width: 44px;
+  height: 22px;  
+}
+
+.form-applicaion-labeljobrole {
+  grid-area: labeljobrole;
+}
+
+.form-applicaion-jobrole {
+  grid-area: jobrole;
+  width: 460px;
+  height: 64px;
+  padding: 0 15px;
+  border-radius: 3px;   
+  border: 1px solid #D2D2D2; 
+  appearance: none;
+}
+
+option {
+  font-family: Calibri;
```

Comment:
please use the family as a fallback as well in all fonts. for example, font-family: Calibri, sans-serif